proposals/cloudevents.md

```diff
@@ -0,0 +1,154 @@
+# CloudEvents
+
+**Name of project**: CloudEvents
+
+**Description**:
+
+Last year the CNCF TOC created the Serverless Working Group to investigate
+the Serverless landscape. The outputs of the WG included:
+- a [whitepaper](https://github.com/cncf/wg-serverless#whitepaper) that:
+  - defines Serverless and its terminology
+  - describes common use cases for the technology
+  - compares it with other Cloud Native technologies and **aaS environments
+  - describes the common architecture of Serverless platforms
+- a [landscape document](https://docs.google.com/spreadsheets/d/10rSQ8rMhYDgf_ib3n6kfzwEuoE88qr0amUPRxKbwVCk/edit#gid=0)
+  that lists well-known open-source and proprietary Serverless platforms
+  and tools
+- a set of recommended next steps for the WG, as part of the whitepaper:
+  - encourage more serverless technology vendors and open source developers
```

Comment:
Are we capitalizing serverless all the time? This is inconsistent with previous usage above, but I don't know the idiom here.

Reply:
I went with making it "S" :-)